foo.rb

```diff
@@ -1,6 +1,5 @@
 class Block
   def foo
-    puts "one"
-    yield
```

Comment:
this passes with no other changes: `assert_output("one\ntwo\n") { b.foo { puts "two" } }`